src/jit/emitxarch.cpp

```diff
@@ -6522,10 +6527,16 @@ void emitter::emitIns_SIMD_R_R_S_R(
 
 void emitter::emitIns_S(instruction ins, emitAttr attr, int varx, int offs)
 {
+    UNATIVE_OFFSET sz;
     instrDesc*     id  = emitNewInstr(attr);
-    UNATIVE_OFFSET sz  = emitInsSizeSV(id, insCodeMR(ins), varx, offs);
     insFormat      fmt = emitInsModeFormat(ins, IF_SRD);
 
+    id->idIns(ins);
```

Comment:
These ones were the ones that were calling `emitInsSize*` without initialize `id->idIns`